tmt-app/src/constants/careerMajorList.js

```diff
@@ -6,96 +6,112 @@ export const CAREERS_LIST = [
     name: "Accounting/Finance",
     desc: "Accounting is the process of recording, classifying, and summarizing financial transactions to provide information that is useful in making business decisions. Finance is the study of how individuals and organizations manage their money, including the processes of acquiring and using financial resources over time. Accountants and financial professionals use tools such as financial statements and spreadsheets to analyze and interpret financial data, and they may also be involved in forecasting and budgeting, risk management, and decision-making",
     link: "https://www.zippia.com/accountant-jobs/",
+    key: "account-finance",
```

Comment:
If we're putting keys in this array of objects, I think we should just make this an object. It fits the key, value use case